src/components/NavBar/NavBar.jsx

```diff
@@ -55,7 +55,7 @@ const NavBar = () => {
   };
 
   // Event handler for logging out of SOLID POD and removing items from localStorage
-  const handleLogout = () => {
+  const handle = () => {
```

Comment:
same goes here. There's no good benefit in deleting "Logout" here.   Now, "what is it handling? I don't know, I have to go dig around to find out." 

and if you DID think this was a good idea, why only do it with this modal and it's handle function?  There are a lot more handle functions in this file, but you left them specific.